db/migrate/20171002234500_create_taxon_descriptions.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTaxonDescriptions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :taxon_descriptions do |t|
+      t.bigint :version
```

Comment:
maybe you can actually add a description field here?  which will have descriptions like "download from 10/2/2017 ncbi"